README.md

```diff
@@ -0,0 +1,185 @@
+Topology
+========
+
+
+This repository contains information for known projects, site resources, and
+Virtual Organizations known to the OSG, as well as scripts for managing and
+displaying this information.  The information was sourced from the old OIM site
+that was hosted at oim.grid.iu.edu.
+
+
+Structure of the Data
+---------------------
+
+The data is organized as files in [YAML format](https://en.wikipedia.org/wiki/YAML)
+as follows:
+
+-   `projects/` contains information about research projects that run jobs on OSG.
+    Each project has its own file, named `<PROJECT>.yaml`
+
+-   `virtual-organizations/` contains information about Virtual Organizations
+    (VOs).
+    Each VO has its own file, named `<VO>.yaml`.  In addition, there is a file
+    containing information about "reporting groups" called
```

Comment:
We should try and flesh out some sort of definition for reporting groups

Reply:
I hope someone can tell me the definition of a reporting group.

Reply:
Marina and Rob are still around, one of them may know the anwer